<ui version="4.0" >
 <class>EditorFormClass</class>
 <widget class="QWidget" name="EditorFormClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>EditorForm</string>
  </property>
  <widget class="QGraphicsView" name="editorView" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>388</width>
     <height>240</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy" >
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy" >
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="dragMode" >
    <enum>QGraphicsView::NoDrag</enum>
   </property>
   <property name="transformationAnchor" >
    <enum>QGraphicsView::NoAnchor</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="undoButton" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>290</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="redoButton" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>290</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="widthEdit" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>290</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="undoViewButton" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>290</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>UV</string>
   </property>
  </widget>
  <widget class="QToolButton" name="penToolButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Pen</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="autoExclusive" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="svgToolButton" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>10</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>SVG</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="autoExclusive" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="pixmapToolButton" >
   <property name="geometry" >
    <rect>
     <x>350</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Photo</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="autoExclusive" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="saveMetaButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>360</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save Meta</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pasteButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>290</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Paste</string>
   </property>
  </widget>
  <widget class="QLabel" name="cacheLabel" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>330</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cache</string>
   </property>
  </widget>
  <widget class="QToolButton" name="savePNGButton" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>360</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save PNG</string>
   </property>
  </widget>
  <widget class="QToolButton" name="clearButton" >
   <property name="geometry" >
    <rect>
     <x>420</x>
     <y>10</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Clear scene</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filenameEdit" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>360</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>text.png</string>
   </property>
  </widget>
  <widget class="QToolButton" name="textToolButton" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Text</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="autoExclusive" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fontSizeBox" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>320</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="minimum" >
    <double>4.000000000000000</double>
   </property>
   <property name="value" >
    <double>4.000000000000000</double>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>320</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="boldButton" >
   <property name="geometry" >
    <rect>
     <x>354</x>
     <y>320</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string>B</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="oilBrushToolButton" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Oil brush</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="autoExclusive" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="eraserToolButton" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Eraser</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="autoExclusive" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="sprayToolButton" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Spray</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="autoExclusive" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="PaletteWidget" native="1" name="colorPalette" >
   <property name="geometry" >
    <rect>
     <x>410</x>
     <y>50</y>
     <width>51</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="paletteManagerButton" >
   <property name="geometry" >
    <rect>
     <x>410</x>
     <y>220</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="arrowType" >
    <enum>Qt::RightArrow</enum>
   </property>
  </widget>
  <widget class="BrushSizeButton" name="penSizeButton" >
   <property name="geometry" >
    <rect>
     <x>410</x>
     <y>250</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
   <property name="iconSize" >
    <size>
     <width>58</width>
     <height>58</height>
    </size>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="popupMode" >
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
  </widget>
  <widget class="BrushAlphaButton" name="penAlphaButton" >
   <property name="geometry" >
    <rect>
     <x>410</x>
     <y>320</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
   <property name="iconSize" >
    <size>
     <width>58</width>
     <height>58</height>
    </size>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="popupMode" >
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="simpleBrushToolButton" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Simple</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="autoExclusive" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="bgButton" >
   <property name="geometry" >
    <rect>
     <x>350</x>
     <y>290</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string>BG</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="metaFilenameEdit" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>360</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>text.xml</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>PaletteWidget</class>
   <extends>QWidget</extends>
   <header>palettewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BrushSizeButton</class>
   <extends>QToolButton</extends>
   <header>brushstylebutton.h</header>
  </customwidget>
  <customwidget>
   <class>BrushAlphaButton</class>
   <extends>QToolButton</extends>
   <header>brushstylebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
